common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1127,25 +1127,25 @@ def pretty_print_seconds(self, num_seconds):
         Returns time duration nicely formated, e.g. "3 minutes 4 seconds"
         """
         # Here _ is the N variant ungettext that does pluralization with a 3-arg call
-        _ = self.runtime.service(self, "i18n").ungettext
+        ungettext = self.runtime.service(self, "i18n").ungettext
```

Comment:
@nparlante this is a line git blame says you wrote. Anyway. You can only alias `_` to `gettext` or `ugettext`. i18n happens in two steps, a static parser/scraper and a dynamic translation lookup. When you alias `_` to something other than `ugettext`, the static scraper doesn't know that and treats it as `ugettext`.